crates/languages/src/python.rs

```diff
@@ -306,7 +306,18 @@ impl LspAdapter for PythonLspAdapter {
             let mut user_settings =
                 language_server_settings(adapter.as_ref(), &Self::SERVER_NAME, cx)
                     .and_then(|s| s.settings.clone())
-                    .unwrap_or_default();
+                    .unwrap_or_else(|| {
+                        json!({
+                            "plugins": {
+                                "pycodestyle": {"enabled": false},
+                                "rope_autoimport": {"enabled": true, "memory": true},
+                                "pylsp_mypy": {"enabled": false}
+                            },
+                            "rope": {
+                                "ropeFolder": null
+                            },
+                        })
+                    });
```

Comment:
Let's revert this bit. We should not be applying pylsp settings to PyRight (they're two different language servers, tho admittedly we have both implementations in a single source file).
```suggestion
                    .unwrap_or_default();
```